pixl_dcmd/pyproject.toml

```diff
@@ -10,7 +10,7 @@ dependencies = [
     "core==0.2.0rc0",
     "arrow==1.3.0",
     "dicom-anonymizer==1.0.12",
-    "dicom-validator==0.6.2",
+    "dicom-validator==0.6.3",
```

Comment:
we were running into this issue with 0.6.2 https://github.com/pydicom/dicom-validator/pull/123

Reply:
ah great to see its fixed